distribution/src/conf/deployment.toml

```diff
@@ -1,8 +1,49 @@
 [server]
 hostname = "localhost"
+# offset  = 0
+# hide_admin_service_wsdl = true
+# proxy_context_path = ""
+# hot_deployment = true
+# enableMTOM = false
+# enableSwA = false
+# userAgent = "WSO2 Micro Integrator 1.0.0"
+# serverDetails = "WSO2 Micro Integrator 1.0.0"
```

Comment:
Is it possible to have the actual server version here?

Reply:
IMO we don't need these here